generatebundlefile/hack/release_prod.sh

```diff
@@ -44,33 +44,19 @@ credential_source=EcsContainer
 EOF
 
 export AWS_CONFIG_FILE=${BASE_DIRECTORY}/generatebundlefile/prodconfigfile
```

Comment:
Do we still need to keep this config file? we are not assuming any role in a different account, so we can probably remove this and the codeblocks that are executed when the profile is not default.

Reply:
Updated